src/driver/spi_bmi270.c

```diff
@@ -59,6 +69,49 @@ void bmi270_configure() {
   bmi270_write(BMI270_REG_INIT_CTRL, 0x1);
   time_delay_ms(1);
 
+  bmi270_write(BMI270_REG_PWR_CONF, 0x0);
+  time_delay_ms(10);
+
+  bmi270_write(BMI270_REG_OFFSET_6, 0x80);
+  time_delay_ms(10);
+
+  bmi270_write(BMI270_REG_PWR_CTRL, 0x04);
+  time_delay_ms(10);
+
+  bmi270_write(BMI270_REG_GYR_CRT_CONF, 0x04);
+  time_delay_ms(10);
+
+  bmi270_write(BMI270_REG_FEAT_PAGE, 0x01);
+  time_delay_ms(10);
+
+  bmi270_write16(BMI270_REG_FEATURES_1_G_TRIG_1, 0x0100);
+  time_delay_ms(10);
+
+  bmi270_write(BMI270_REG_CMD, 0x02);
+  time_delay_ms(5000);
+
+  uint8_t crt_check = bmi270_read(BMI270_REG_GYR_CRT_CONF);
+  uint8_t crt_check_count = 0;
+  time_delay_ms(1000);
+
+  while ((crt_check != 0x00) && (crt_check_count++ < 5)) {
+    crt_check = bmi270_read(BMI270_REG_GYR_CRT_CONF);
```

Comment:
Please put a breakpoint at this line and check the read result for the `BMI270_REG_GYR_CRT_CONF`, should be `0x00` if CRT done.

Reply:
will do.